lib/resources/addresses.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-var util         = require('util');
-var ResourceBase = require('./resourceBase');
+const util         = require('util');
```

Comment:
nit: alphabetize and capitalize

Reply:
using es6 classes is going to get rid of the `util`, so should I just leave it for now?

Reply:
oh ok then yeah